### DISCUSSION TITLE
Security Onion Duplicate Acks and TCP retransmits

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#8307] [General] [answered]
Good Afternoon all,

I am searching the internet, but haven't found an answer.  

Are we able to use Security Onion / Zeek to identify duplication acks and or TCP retransmits?

Thanks for the help.

**Comment by btdewey** [comment, marked as answer]
That's a pretty low level analysis.  The zeek weird log has some things like that, but I'm not certain if it specifically has duplicate acks.  You can extract out pcaps and drop them into wireshark, but that's a pretty manual analysis.  Suricata will alert on some local subnet based attacks.  Are you looking for a specific type of attack or just looking for issues on the network?

**Reply by oneCrazyAdmin** [reply]
I am looking for issues on the network.  We are tracking down an issue.  We are running PCAPs at the end points, but was hoping that we could see something with our SO sensor.

**Comment by btdewey** [comment]
In theory you should be able to get a date/time range of pcaps from the sensor, but I know my team has struggled a little with that.  Have you tried this from the documentation?  I've been working with the playbook and ingest portions of the platform so I haven't done much other than download a specific stream from an event.
https://docs.securityonion.net/en/2.3/pcap.html

**Reply by oneCrazyAdmin** [reply]
I have tried that in the past.  I haven't had much luck with it.  Usually, I go to an event and do a PCAP from there.

I will check out the docs and see if they can help.
